AppKit/CPTableView.j

```diff
@@ -6400,18 +6400,12 @@ var CPTableViewDataSourceKey                = @"CPTableViewDataSourceKey",
 
 - (BOOL)setThemeState:(ThemeState)aState
 {
-    if (aState.isa && [aState isKindOfClass:CPArray])
```

Comment:
Yeah, I was thinking about that but this is part of the speedup. These methods are called 100,000 times when loading a large cib file and this check takes a lot of time.
I think it is enough to do the check in ```Debug``` and throw an exception as is done now in the superclass. This will give a clear description what is wrong and also give the speed when running in ```Release```